basic_integral/basic_integral.cpp

```diff
@@ -0,0 +1,24 @@
+#include <iostream>
+#include <math.h>
```

Comment:
Any particular reason for using the .h file? cmath is standard c++ library which I guess is more reliable.